src/common/utils/form-values/form-values.test.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  formElementsMock,
+  formValuesMock,
+  handleFormMock,
+} from 'tests/mocks/common-configurations'
+import { getFormValues } from './form-values'
+
+function Form() {
+  const { form, holderNameInput, cvvInput, expirationDateInput, numberInput } =
+    handleFormMock()
+
+  form.setAttribute(formElementsMock.form, '')
+  holderNameInput.setAttribute(formElementsMock.holderName, '')
+  numberInput.setAttribute(formElementsMock.number, '')
+  cvvInput.setAttribute(formElementsMock.cvv, '')
+  expirationDateInput.setAttribute(formElementsMock.expirationDate, '')
+
+  document.body.appendChild(form)
+  form.appendChild(holderNameInput)
+  form.appendChild(numberInput)
+  form.appendChild(expirationDateInput)
+  form.appendChild(cvvInput)
+}
+describe('getFormValues', () => {
```

Comment:
Ajustar para seguir com aquela nossa linha de contexto em arquivos que só exportam funções

```suggestion
describe('form-values', () => {
  describe('getFormValues', () => {
})
```